test/unittests/t_uuid.cc

```diff
@@ -62,7 +62,7 @@ TEST(T_Uuid, FailRead) {
   fclose(f);
   UnlinkGuard unlink_guard(path);
   UniquePtr<Uuid> uuid(Uuid::Create(path));
-  EXPECT_TRUE(uuid == NULL);
+  EXPECT_TRUE(uuid.IsValid());
```

Comment:
Mhh, reverting the latest fix and re-running the test cases produces errors as expected. Perhaps I mixed up a `--gtest_filter=...`. Sorry for the fuzz.